indexd/index/blueprint.py

```diff
@@ -503,6 +504,28 @@ def get_all_index_record_versions(record):
     return flask.jsonify(ret), 200
 
 
+@blueprint.route("/index/<path:record>/versions", methods=["PUT"])
+def update_all_index_record_versions(record):
+    """
+    Update metadata for all record versions.
+    NOTE currently the only fields that can be updated for all versions are
+    (`authz`, `acl`).
+    """
+    request_json = flask.request.get_json(force=True)
+    try:
+        jsonschema.validate(request_json, UPDATE_ALL_VERSIONS_SCHEMA)
+    except jsonschema.ValidationError as err:
+        logger.warn(f"Bad request body:\n{err}")
+        raise UserError(err)
+
+    acl = request_json.get("acl", None)
+    authz = request_json.get("authz", None)
```

Comment:
you can just use `.get("acl")` because None is the default returned value 👍

Reply:
Word, thanks! More python knowledge 🐍